diff --git a/node_modules/@medusajs/admin-vite-plugin/dist/index.js b/node_modules/@medusajs/admin-vite-plugin/dist/index.js
index aba514e..e832070 100644
--- a/node_modules/@medusajs/admin-vite-plugin/dist/index.js
+++ b/node_modules/@medusajs/admin-vite-plugin/dist/index.js
@@ -192,7 +192,7 @@ function generateHash(content) {
   return import_node_crypto.default.createHash("md5").update(content).digest("hex");
 }
 function isFileInAdminSubdirectory(file, subdirectory) {
-  const normalizedPath = normalizePath(file);
+  const normalizedPath = "/@fs" + normalizePath(file);
   return normalizedPath.includes(`/src/admin/${subdirectory}/`);
 }
 
@@ -447,7 +447,7 @@ function generateCustomFieldConfigName(index) {
   return `CustomFieldConfig${index}`;
 }
 function generateImport(file, index) {
-  const path3 = normalizePath(file);
+  const path3 = "/@fs" + normalizePath(file);
   return `import ${generateCustomFieldConfigName(index)} from "${path3}"`;
 }
 
@@ -605,7 +605,7 @@ function generateCustomFieldConfigName2(index) {
   return `CustomFieldConfig${index}`;
 }
 function generateImport2(file, index) {
-  const path3 = normalizePath(file);
+  const path3 = "/@fs" + normalizePath(file);
   return `import ${generateCustomFieldConfigName2(index)} from "${path3}"`;
 }
 function getForms(path3, model, index, file) {
@@ -1079,7 +1079,7 @@ function generateCustomFieldConfigName3(index) {
   return `CustomFieldConfig${index}`;
 }
 function generateImport3(file, index) {
-  const path3 = normalizePath(file);
+  const path3 = "/@fs" + normalizePath(file);
   return `import ${generateCustomFieldConfigName3(index)} from "${path3}"`;
 }
 function getLink(path3, index, file) {
@@ -1125,7 +1125,7 @@ var import_outdent2 = require("outdent");
 async function generateI18n(sources) {
   const indexFiles = await getI18nIndexFilesFromSources(sources);
   const imports = indexFiles.map((file, index) => {
-    const normalizedPath = normalizePath(file);
+    const normalizedPath = "/@fs" + normalizePath(file);
     return `import i18nTranslations${index} from "${normalizedPath}"`;
   });
   let mergeCode = "{}";
@@ -1152,7 +1152,7 @@ var import_outdent3 = require("outdent");
 
 // src/routes/helpers.ts
 function getRoute(file) {
-  const importPath = normalizePath(file);
+  const importPath = "/@fs" + normalizePath(file);
   return importPath.replace(/.*\/admin\/(routes)/, "").replace("[[*]]", "*?").replace("[*]", "*").replace(/\(([^\[\]\)]+)\)/g, "$1?").replace(/\[\[([^\]]+)\]\]/g, ":$1?").replace(/\[([^\]]+)\]/g, ":$1").replace(
     new RegExp(
       `/page\\.(${VALID_FILE_EXTENSIONS.map((ext) => ext.slice(1)).join(
@@ -1220,7 +1220,7 @@ async function parseFile3(file, index) {
   };
 }
 function generateImport4(file, index) {
-  const path3 = normalizePath(file);
+  const path3 = "/@fs" + normalizePath(file);
   return `import { config as ${generateRouteConfigName(index)} } from "${path3}"`;
 }
 function generateMenuItem(config, file, index) {
@@ -1515,7 +1515,7 @@ ${e}`
 function generateImports(file, index, hasHandle, hasLoader) {
   const imports = [];
   const route = generateRouteComponentName(index);
-  const importPath = normalizePath(file);
+  const importPath = "/@fs" + normalizePath(file);
   if (!hasHandle && !hasLoader) {
     imports.push(`import ${route} from "${importPath}"`);
   } else {
@@ -1775,7 +1775,7 @@ function generateWidgetConfigName(index) {
   return `WidgetConfig${index}`;
 }
 function generateImport5(file, index) {
-  const path3 = normalizePath(file);
+  const path3 = "/@fs" + normalizePath(file);
   return `import ${generateWidgetComponentName(
     index
   )}, { config as ${generateWidgetConfigName(index)} } from "${path3}"`;
diff --git a/node_modules/@medusajs/admin-vite-plugin/dist/index.mjs b/node_modules/@medusajs/admin-vite-plugin/dist/index.mjs
index 8b4a2b5..26e032a 100644
--- a/node_modules/@medusajs/admin-vite-plugin/dist/index.mjs
+++ b/node_modules/@medusajs/admin-vite-plugin/dist/index.mjs
@@ -174,7 +174,7 @@ function generateHash(content) {
   return crypto.createHash("md5").update(content).digest("hex");
 }
 function isFileInAdminSubdirectory(file, subdirectory) {
-  const normalizedPath = normalizePath(file);
+  const normalizedPath = "/@fs" + normalizePath(file);
   return normalizedPath.includes(`/src/admin/${subdirectory}/`);
 }
 
@@ -431,7 +431,7 @@ function generateCustomFieldConfigName(index) {
   return `CustomFieldConfig${index}`;
 }
 function generateImport(file, index) {
-  const path3 = normalizePath(file);
+  const path3 = "/@fs" + normalizePath(file);
   return `import ${generateCustomFieldConfigName(index)} from "${path3}"`;
 }
 
@@ -594,7 +594,7 @@ function generateCustomFieldConfigName2(index) {
   return `CustomFieldConfig${index}`;
 }
 function generateImport2(file, index) {
-  const path3 = normalizePath(file);
+  const path3 = "/@fs" + normalizePath(file);
   return `import ${generateCustomFieldConfigName2(index)} from "${path3}"`;
 }
 function getForms(path3, model, index, file) {
@@ -1068,7 +1068,7 @@ function generateCustomFieldConfigName3(index) {
   return `CustomFieldConfig${index}`;
 }
 function generateImport3(file, index) {
-  const path3 = normalizePath(file);
+  const path3 = "/@fs" + normalizePath(file);
   return `import ${generateCustomFieldConfigName3(index)} from "${path3}"`;
 }
 function getLink(path3, index, file) {
@@ -1114,7 +1114,7 @@ import { outdent as outdent2 } from "outdent";
 async function generateI18n(sources) {
   const indexFiles = await getI18nIndexFilesFromSources(sources);
   const imports = indexFiles.map((file, index) => {
-    const normalizedPath = normalizePath(file);
+    const normalizedPath = "/@fs" + normalizePath(file);
     return `import i18nTranslations${index} from "${normalizedPath}"`;
   });
   let mergeCode = "{}";
@@ -1143,7 +1143,7 @@ import { outdent as outdent3 } from "outdent";
 
 // src/routes/helpers.ts
 function getRoute(file) {
-  const importPath = normalizePath(file);
+  const importPath = "/@fs" + normalizePath(file);
   return importPath.replace(/.*\/admin\/(routes)/, "").replace("[[*]]", "*?").replace("[*]", "*").replace(/\(([^\[\]\)]+)\)/g, "$1?").replace(/\[\[([^\]]+)\]\]/g, ":$1?").replace(/\[([^\]]+)\]/g, ":$1").replace(
     new RegExp(
       `/page\\.(${VALID_FILE_EXTENSIONS.map((ext) => ext.slice(1)).join(
@@ -1211,7 +1211,7 @@ async function parseFile3(file, index) {
   };
 }
 function generateImport4(file, index) {
-  const path3 = normalizePath(file);
+  const path3 = "/@fs" + normalizePath(file);
   return `import { config as ${generateRouteConfigName(index)} } from "${path3}"`;
 }
 function generateMenuItem(config, file, index) {
@@ -1506,7 +1506,7 @@ ${e}`
 function generateImports(file, index, hasHandle, hasLoader) {
   const imports = [];
   const route = generateRouteComponentName(index);
-  const importPath = normalizePath(file);
+  const importPath = "/@fs" + normalizePath(file);
   if (!hasHandle && !hasLoader) {
     imports.push(`import ${route} from "${importPath}"`);
   } else {
@@ -1766,7 +1766,7 @@ function generateWidgetConfigName(index) {
   return `WidgetConfig${index}`;
 }
 function generateImport5(file, index) {
-  const path3 = normalizePath(file);
+  const path3 = "/@fs" + normalizePath(file);
   return `import ${generateWidgetComponentName(
     index
   )}, { config as ${generateWidgetConfigName(index)} } from "${path3}"`;
